_includes/specials/bech32/21-brd.md

```diff
@@ -0,0 +1,83 @@
+
+*The following case study contributed by Optech member company [BRD][]
+describes what they learned implementing bech32 and other segwit
+technology for their wallet.*
+
+BRD Wallet began implementing bech32 support in January 2018 with the
+[addition of bech32 decoding and encoding support][brd pr1] to
+[breadwallet-core][], an MIT-licensed cross-platform C library
+with no external dependencies.  All of BRD's software avoids
+third-party library dependencies as much as possible, with it currently
+using only Pieter Wuille's [libsecp256k1][].  Minimizing dependencies is
+typical for a high-security crypto project. For the bech32
+implementation, we found that bech32's [BIP173][] is pretty well
```

Comment:
Curious shift here, and a second time in line 72, from third-person to first-person narrative and back. Particularly this first one as the case study begins in the third person.